__TESTS__/unit/toJson/backgroundRemoval.toJson.test.ts

```diff
@@ -0,0 +1,19 @@
+import {Transformation} from "../../../src";
+import {ForegroundObject} from "../../../src/qualifiers/foregroundObject";
+import {Effect} from "../../../src/actions/effect";
+
+describe('BackgroundRemoval toJson()', () => {
+  it('with fineEdges and hints', () => {
+    const transformation = new Transformation()
+      .addAction(Effect.backgroundRemoval(true, [ForegroundObject.CAT, ForegroundObject.DOG]));
```

Comment:
The syntax design question: do we want to have apply-style methods like `.fineEdges(true).hints([ForegroundObject.CAT, ForegroundObject.DOG])` instead of passing args in the constructor?